### PR TITLE
Feat/oslog

### PR DESCRIPTION
**Pull request by rathishubham7**

### CODE REVIEW

**Comment by rathishubham7** [comment]
Remove the old logs

File: Sources/TorusUtils/Extensions/TorusUtils+extension.swift [hunk 1]
```diff
@@ -129,7 +133,8 @@ extension TorusUtils {
                 rpcdata = try JSONSerialization.data(withJSONObject: dataForRequest)
             }
         } catch {
-            self.logger.error("retrieveDecryptAndReconstruct - error:", error)
+            os_log("retrieveDecryptAndReconstruct - error: %@", log: Log.core, type: .error, error.localizedDescription)
+            // self.logger.error("retrieveDecryptAndReconstruct - error:", error)
```
Comment (rathishubham7) [inline Sources/TorusUtils/Extensions/TorusUtils+extension.swift:137]: remove self.logger comments

**Comment by michaellee8** [comment]
@rathishubham7 resolved conflict on Package.swift, also removed bestlogger from podspec too? looks good for you?

**Comment by michaellee8** [comment]
@rathishubham7 please review

**Pull request merged**